### ISSUE TITLE
Camera Button Screen Off 

### ISSUE THREAD

**Issue by artisak2** [keymapperorg/KeyMapper#911]
Requesting enhancement to support camera button (27) detection with screen off.

Also Focus button (80) with screen off

**Comment by GL513** [comment]
Is your device rooted? Alot of things might not work if keymapper doesn't have root access.

https://docs.keymapper.club/includes/trigger-options/trigger-when-screen-off/

Without root access, keymapper currently can't trigger with the screen off.

I have also noticed that on certain keyboards you can't remap the "lock"/"power off" buttons, so this might be a similar case.

**Comment by artisak2** [comment]
Yes device is rooted. Confirmed that volume keys are detected with screen off.
But, like in the link you supplied, it says the only current supported buttons are for volume ud/dn and headset buttons.
It also mentions to let developer know any buttons we would like supported.

**Comment by sds100** [comment]
@artisak2 can you run the `adb shell getevent -lq` command, press those buttons and then post the output.

**Comment by artisak2** [comment]
`/dev/input/event3: EV_KEY       02fe                 DOWN
/dev/input/event3: EV_SYN       SYN_REPORT           00000000
/dev/input/event3: EV_KEY       KEY_CAMERA_FOCUS     DOWN
/dev/input/event3: EV_SYN       SYN_REPORT           00000000
/dev/input/event3: EV_KEY       02fe                 UP
/dev/input/event3: EV_SYN       SYN_REPORT           00000000
/dev/input/event3: EV_KEY       KEY_CAMERA_FOCUS     UP
/dev/input/event3: EV_SYN       SYN_REPORT           00000000`
